app/views/automobiles/show.html.haml

```diff
@@ -0,0 +1,12 @@
+.flex.justify-center.mt-8.mr-8.ml-8
+    .max-w-lg.rounded.overflow-hidden.shadow-lg{id: dom_id(@automobile)}
+        %img.w-full{src: "https://www.christenseninstitute.org/wp-content/uploads/2016/08/nonconsumption-ford-model-t-1.jpg"}/
+        .px-6.py-4
+            .font-bold.text-xl.mb-2= "#{@automobile.brand} #{@automobile.model}"
+            %p.text-grey-darker.text-base
+            "Description"
```

Comment:
So this feature says that you want to have a show page to be able to see more information about the car. You are hard coding the `"Description"` here. Add description as an attribute to the car, add it to the background table in the feature test and show it in the view.

Reply:
Yeah...I knew this.  I hardcoded it last night thinking I'll fix it today and I overlooked it.  We'll get on it.

Reply:
Description was just a placeholder because we have not finalized the car api and available fields we will be able to show on the show car page. We have removed the placeholder, and will add the attributes as and when we finalize the api and use it.

Reply:
All should be good and in working order now.